content/spire/try/spire+aws-oidc-federation.md

```diff
@@ -297,12 +308,14 @@ client-74d4467b44-7nrs2   1/1     Running   0          27s
 $ kubectl exec -it client-74d4467b44-7nrs2 /bin/sh
 ```
 
-3. Fetch a JWT SVID from the identity provider on AWS and save the token from the JWT SVID into a file called `token`:
+3. Fetch a JWT SVID from the identity provider on AWS and save the token from the JWT SVID into a file on the client container called `token`:
 
 ```console
-/opt/spire/bin/spire-agent api fetch jwt -audience mys3 -socketPath /run/spire/sockets/agent.sock | sed '2q' | sed 's/[[:space:]]//g' > token
+/opt/spire/bin/spire-agent api fetch jwt -audience mys3 -socketPath /run/spire/sockets/agent.sock | sed '2!d' | sed 's/[[:space:]]//g' > token
 ```
 
+To familiarize yourself with the form of the token 
```

Comment:
is something expected after this sentence?

Reply:
Oops, now it is: 
To familiarize yourself with the form of the token, you may want to view it with the `cat` command.